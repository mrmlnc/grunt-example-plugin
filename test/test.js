var assert = require('assert');
var grunt = require('grunt');

it('Default test', function() {
  var result = grunt.file.read('.tmp/test.html');
  var expected = grunt.file.read('test/expected/test.html');
  assert.equal(result, expected);
});
