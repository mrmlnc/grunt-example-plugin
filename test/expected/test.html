<p><strong>test</strong>
<em>test</em></p>
